docs/gen-ai/llm-spans.md

```diff
@@ -0,0 +1,83 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM Calls
+--->
+
+# Semantic Conventions for LLM requests
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+- [LLM Request attributes](#llm-request-attributes)
+- [Events](#events)
+
+<!-- tocstop -->
+
+A request to an LLM is modeled as a span in a trace.
+
+The **span name** SHOULD be set to a low cardinality value representing the request made to an LLM.
+It MAY be a name of the API endpoint for the LLM being called.
+
+## Configuration
+
+Instrumentations for LLMs MUST offer the ability to turn off capture of prompts and completions. This is for three primary reasons:
+
+1. Data privacy concerns. End users of LLM applications may input sensitive information or personally identifiable information (PII) that they do not wish to be sent to a telemetry backend.
+2. Data size concerns. Although there is no specified limit to sizes, there are practical limitations in programming languages and telemety systems. Some LLMs allow for extremely large context windows that end users may take full advantage of.
+3. Performance concerns. Sending large amounts of data to a telemetry backend may cause performance issues for the application.
+
+By default, these configurations SHOULD NOT capture prompts and completions.
```

Comment:
I think there is a contradiction between this sentence, the requirement level on the corresponding attributes, and the previous paragraph. 

If we want prompts/completions to be off by default, the attributes should have opt-in level. 
Then we previous paragraph should be rephrased to something like

"Instrumentations MAY capture prompts and completions. They contain sensitive and verbose content and therefore are only captured if enabled explicitly - see [Attribute Requirement Levels - Opt-in](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/#opt-in)"  for the details.

I *think* we can entertain the idea of more relaxed opt-in. E.g. if instrumentation can reliably detect that code is running inside IDE or in development environment, etc. Let's see.